src/tribol/interface/tribol.cpp

```diff
@@ -692,6 +692,12 @@ void registerMortarGaps( integer meshId,
                     "on mesh " << meshId << ".");
       mesh.m_isValid = false;
    }
+   else if (gaps == nullptr && mesh.m_numCells <= 0)
```

Comment:
Should this be >= 0?

Reply:
the first check is null pointer + non-null mesh (mesh.m_numCells > 0) and we issue a warning. The else if checks for a null pointer and a null mesh (this should be allowed) (mesh.m_numCells <=0) and issues a debug statement (arguable don't need to issue anything, but might be nice for debug.

Reply:
OK i see it now, I missed that these are new checks (hard to see on GitHub).

Reply:
this has been resolved per outside discussion.